### PR TITLE
Add the UI

### PR DESCRIPTION
**Pull request by alicemirigo92**
- Build interactive console apps.
- Run a program using the command line.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    end
+  end
+
+  def list_all_people
+    puts 'List of all people'
+    @people.each do |person|
+      puts "[#{person.class}] name: #{person.name}, ID: #{person.id}, age: #{person.age}\n"
+      puts
+    end
+  end
```
Comment (Owusu-Desmond) [inline app.rb:63]: - [ ] Please your app can not list all people so make sure you fix this error :point_down: to get this method of listing people to work perfectly. :pray: 
  
  image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    puts "#{book.title} has been created successfully.\n"
+    puts
+  end
+
+  def create_rental
+    book = select_book
+    person = select_person
+    rental_date = enter_rental_date
+    create_rental_record(rental_date, book, person)
+  end
```
Comment (Owusu-Desmond) [inline app.rb]: - [ ] Please your app can not create a rental so make sure you fix this error :point_down: to get this method of creating a rental to work perfectly. :pray: 
  
image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @alicemirigo92 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (rloterh) [reply app.rb:63]: well resolved! ✔️

File: app.rb (hunk 2 above)
Comment (rloterh) [reply app.rb]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Gathoni,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉 💯 🌟 

## Highlights 👍
- Interactive console app is working as intended!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- Kindly ensure that the _Getting Started_ section_ is filled with the appropriate instructions on how to _setup_ and _run_ your app.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers..Happy coding! 💻  🍷  🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**